### DISCUSSION TITLE
关于DatePicker

### DISCUSSION THREAD

**Discussion by jkiss** [ant-design/ant-design-mobile#3911] [Q&A] [answered]
DatePicker默认三列 “年月日”，但是我只想要两列 “年月”，没找到怎么设置，还是没有这个功能呢？

**Comment by awmleer** [comment]
没有这个功能的，DatePicker 的 value 和 onChange 都是 `Date` 类型的对象

如果想选“年+月”，可以自己直接用 Picker 实现，其实很简单的

**Comment by jkiss** [comment, marked as answer]
目前DatePicker已具备控制精度功能：[控制选择精度](https://next.mobile.ant.design/components/date-picker)

**Reply by awmleer** [reply]
感谢更新～
